### PR TITLE
add images for nodejs version 14

### PR DESCRIPTION
**Pull request by pkubatrh**

### CODE REVIEW

File: 14/Dockerfile.fedora [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      io.s2i.scripts-url="image:///usr/libexec/s2i" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="${APP_ROOT}/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858"\
+      com.redhat.component="$NAME" \
+      name="$FGC/$NAME" \
+      version="$NODEJS_VERSION" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/sclorg/s2i-nodejs-container" \
+      usage="s2i build <SOURCE-REPOSITORY> centos/$NAME-$NODEJS_VERSION-centos7:latest <APP-NAME>"
```
[the remark this answers is not in the document]
Comment (pkubatrh) [reply 14/Dockerfile.fedora:49]: Definitely not. probably best thing to remove it altogether.

File: 14/Dockerfile.fedora [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+# TODO: Add nodejs-nodemon into Fedora
```
[the remark this answers is not in the document]
Comment (pkubatrh) [reply 14/Dockerfile.fedora:51]: Will check and add it if it can be used

File: 14/s2i/bin/usage [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+DISTRO=`cat /etc/*-release | grep ^ID= | grep -Po '".*?"' | tr -d '"'`
+
+cat <<EOF
+This is a S2I nodejs-8 ${DISTRO} base image:
```
[the remark this answers is not in the document]
Comment (pkubatrh) [reply 14/s2i/bin/usage:6]: Ah, only checked for version 10 strings, so missed leftovers from version 8. Thanks for catching this. Will fix for other version as well if needed.

File: 14/s2i/bin/usage [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+To use it, install S2I: https://github.com/openshift/source-to-image
+
+Sample invocation:
+
+s2i build https://github.com/sclorg/s2i-nodejs-container.git --context-dir=8/test/test-app/ ${NAMESPACE}/nodejs-8-${DISTRO}7 nodejs-sample-app
```
[the remark this answers is not in the document]
Comment (pkubatrh) [reply 14/s2i/bin/usage:11]: Same here

**Comment by pkubatrh** [comment]
Seems like we have a limitation of only being able to have one image version per Fedora release, else the higher version overrides the lower one :/

```
14:36:41 -> Tagging image 'e1159a781b9e' as 'f32/nodejs:10' and 'f32/nodejs:latest' and 'f32/nodejs:20200720-d63b187'
14:36:41 -> Tagging image '760e0beaa931' as 'f32/nodejs:12' and 'f32/nodejs:latest' and 'f32/nodejs:20200720-d63b187'
14:36:41 -> Tagging image 'e4753c60f782' as 'f33/nodejs:14' and 'f33/nodejs:latest' and 'f33/nodejs:20200720-d63b187'
```

File: 14/Dockerfile.fedora (hunk 1 above)
[the remark this answers is not in the document]
Comment (hhorak) [reply 14/Dockerfile.fedora:49]: Or use the `oc new-app` example.

**Comment by pkubatrh** [comment]
New changes pushed. Will create a tracker for the issue mentioned above and downgrade node 10 to fc31 base image for now.

File: 14/s2i/bin/assemble [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+# Set the DEV_MODE to false by default.
+if [ -z "$DEV_MODE" ]; then
+  export DEV_MODE=false
+fi
+
+# TODO: Add nodejs-nodemon into Fedora
+# https://bugzilla.redhat.com/show_bug.cgi?id=1433784
+# meanwhile install it during s2i assemble step
+if [ "$(source /etc/os-release ; echo $NAME)" == "Fedora" ] ; then
+  NPM_CONFIG_PREFIX=$HOME/../ npm install -g nodemon && \
```
Comment (phracek) [inline 14/s2i/bin/assemble:75]: nodemon could be a part of Dockerfile.fedore, right? Please move it.

**Comment by phracek** [comment]
Sorry for noise regarding imagestreams. If .exclude-{OS} are present, thank tests is skipped.
Then only nodemon issue.

**Comment by pkubatrh** [comment]
[test]

**Pull request merged**